README.md

```diff
@@ -343,7 +343,7 @@ sudo install -m 755 kubeseal-$GOOS-$GOARCH /usr/local/bin/kubeseal
 Now grab your pub-cert.pem file from your cluster, or use the official [OpenFaaS Cloud certificate](https://github.com/openfaas/cloud-functions/blob/master/pub-cert.pem).
 
 ```
-$ kubeseal --fetch-cert > pub-cert.pem
+$ kubeseal --fetch-cert --controller-name ofc-sealedsecrets-sealed-secrets > pub-cert.pem
```

Comment:
Please can you put this on a separate line saying

```
# When SealedSecrets is installed by ofc-bootstrap
```

Reply:
Might be better to add a note about: https://github.com/openfaas-incubator/ofc-bootstrap/blob/master/scripts/export-sealed-secret-pubcert.sh

This does everything automatically ^